### PR TITLE
fix task resource table

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
Signed-off-by: Alexey Chernyshov <alexey.n.chernyshov@gmail.com>

### Description of the Change
Fix resource table order.

### CODE REVIEW

File: core/sector_storage/impl/scheduler_impl.hpp [hunk 1]
```diff
@@ -56,10 +57,28 @@ namespace fc::sector_storage {
...
+        {primitives::kTTProveReplicaUpdate1, 6},
+        {primitives::kTTProveReplicaUpdate2, 7},
+        {primitives::kTTReplicaUpdate, 8},
+        {primitives::kTTAddPiece, 9},
+        {primitives::kTTRegenSectorKey, 10}};
+    const auto left_order = common::getOrDefault(order, lhs.task_type, 0);
+    const auto right_order = common::getOrDefault(order, rhs.task_type, 0);
+
+    return less(lhs.priority,
+                rhs.priority,
```
Comment (Alexey-N-Chernyshov) [inline core/sector_storage/impl/scheduler_impl.hpp:79]: It seemes to be a bug in order previously.

File: core/sector_storage/impl/scheduler_impl.hpp [hunk 2]
```diff
@@ -56,10 +57,28 @@ namespace fc::sector_storage {
   };
 
   inline bool operator<(const TaskRequest &lhs, const TaskRequest &rhs) {
-    return less(rhs.priority,
-                lhs.priority,
-                lhs.task_type,
-                rhs.task_type,
+    // Get TaskType order if present. For TaskTypes not present order is 0.
+    static std::map<TaskType, int> order = {
+        {primitives::kTTFinalize, -2},
```
Comment (Alexey-N-Chernyshov) [inline core/sector_storage/impl/scheduler_impl.hpp:62]: I don't like place for order in Scheduler, but it is used only here.

File: core/primitives/seal_tasks/task.hpp [hunk 3]
```diff
@@ -37,25 +31,5 @@ namespace fc::primitives {
...
-        {kTTPreCommit1, 5},
-        {kTTProveReplicaUpdate1, 6},
-        {kTTProveReplicaUpdate2, 7},
-        {kTTReplicaUpdate, 8},
-        {kTTAddPiece, 9},
-        {kTTRegenSectorKey, 10}};
-
-    return order[lhs] < order[rhs];
-  }
+  const TaskType kTTFinalizeReplicaUpdate("seal/v0/finalize/replicaupdate");
```
Comment (ortyomka) [inline core/primitives/seal_tasks/task.hpp:34]:
```suggestion
  const TaskType kTTFinalizeReplicaUpdate("seal/v0/finalize/replicaupdate");
  
  inline int getPiority(std::string_view task) {
    static std::unordered_map<std::string_view, int> order = {
        {kTTFinalize, -2},
        {kTTFetch, -1},
        {kTTUnseal, 1},
        {kTTCommit1, 2},
        {kTTCommit2, 3},
        {kTTPreCommit2, 4},
        {kTTPreCommit1, 5},
        {kTTProveReplicaUpdate1, 6},
        {kTTProveReplicaUpdate2, 7},
        {kTTReplicaUpdate, 8},
        {kTTAddPiece, 9},
        {kTTRegenSectorKey, 10}};

    return common::getOrDefault(order, task, 0);
  }
```

File: core/sector_storage/impl/scheduler_impl.hpp [hunk 4]
```diff
@@ -56,10 +57,28 @@ namespace fc::sector_storage {
...
+        {primitives::kTTReplicaUpdate, 8},
+        {primitives::kTTAddPiece, 9},
+        {primitives::kTTRegenSectorKey, 10}};
+    const auto left_order = common::getOrDefault(order, lhs.task_type, 0);
+    const auto right_order = common::getOrDefault(order, rhs.task_type, 0);
+
+    return less(lhs.priority,
+                rhs.priority,
+                left_order,
+                right_order,
```
Comment (ortyomka) [inline core/sector_storage/impl/scheduler_impl.hpp:81]:
```suggestion
                getPiority(lhs.task_type),
                getPiority(rhs.task_type),
```

File: core/sector_storage/impl/scheduler_impl.hpp [hunk 5]
```diff
@@ -56,10 +57,28 @@ namespace fc::sector_storage {
...
+        {primitives::kTTCommit2, 3},
+        {primitives::kTTPreCommit2, 4},
+        {primitives::kTTPreCommit1, 5},
+        {primitives::kTTProveReplicaUpdate1, 6},
+        {primitives::kTTProveReplicaUpdate2, 7},
+        {primitives::kTTReplicaUpdate, 8},
+        {primitives::kTTAddPiece, 9},
+        {primitives::kTTRegenSectorKey, 10}};
+    const auto left_order = common::getOrDefault(order, lhs.task_type, 0);
+    const auto right_order = common::getOrDefault(order, rhs.task_type, 0);
```
Comment (ortyomka) [inline core/sector_storage/impl/scheduler_impl.hpp:76]:
```suggestion
```

**Pull request merged**